### PR TITLE
Homework5 + tools package, homework4 not important fix

### PR DESCRIPTION
**Pull request by yeargor**

### CODE REVIEW

File: src/main/java/Homework5/BusTicketService.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+public class BusTicketService {
+
+    public static void main(String[] args) {
+        try {
+            BusTicketGenerator busticketGenerator = new BusTicketGenerator(30);
+            serializeBusTickets(busticketGenerator.getBusTickets());
+
+            List<BusTicket> busTickets = deserializeBusTickets("busTickets.json");
+
+            for (Object i : busTickets) {
```
Comment (Alexmansar) [inline src/main/java/Homework5/BusTicketService.java:24]: for (Ticket ticket : busTickets) {

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@yeargor Hi good practice with annotations and validations 👍  I have some comments)) just check it please)

**Comment by yeargor** [comment]
> @yeargor please, fix this one image and after that you can merge it)) good job

I have already fixed it in the previous commits
image

**Comment by yeargor** [comment]
> @yeargor
> 
> for (Ticket ticket : busTickets) {
> 
> not Object i..

I'm sorry, but that's what I have in my code after fixes

List<BusTicket> busTickets = busTicketSerializer.deserializeBusTickets("busTickets.json");

            for (BusTicket i : busTickets) {
                System.out.println(i.toString());
            }
            
There we can see, that Im using "BusTicket i : busTickets". Im using BusTicket instead of Ticket, because I have created new package for 5th homework and there Im working with such name of class (BusTicket.java)

**Pull request merged**